nixos/default.nix

```diff
@@ -44,10 +38,12 @@ let
       powerManagement = stub;
       security.pam.usb = stub;
       security.pam.mount = stub;
+      security.pam.oath = stub;
       boot.isContainer = lib.mkOption { default = false; };
       boot.initrd.luks = stub;
       networking.wireless = stub;
       networking.connman = stub;
+      virtualisation.vswitch = stub;
```

Comment:
These're options referenced by the proper subset of modules we bring in via `vendor-modules.nix`; we want to keep their references intact to support evaluation, but for some reason (smaller closures? historical misbehaviour?) have no behaviour beyond placeholding. In this case, the `network-interfaces` module will set `vswitch` config, under certain configuration, but we never use those, so.